### DISCUSSION TITLE
Click on progress bar to change the value (all possible values depending on the location on the bar)

### DISCUSSION THREAD

**Discussion by l0f4r0** [polybar/polybar#2983] [Ideas]
It would be nice to change a value simply by clicking on the progress bar (the value would be evaluated depending on where the user has clicked on the bar so every values are possible).

In my case, I would like to change sound and brightness this way but currently I can only use keyboard shortcuts or scroll over the bar...

**Comment by patrick96** [comment]
See #2562
